<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications &amp; talks | Aline Potiron</title> <meta name="author" content="Aline Potiron"> <meta name="description" content="Publications by categories in reversed chronological order."> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.4.0/css/all.min.css" integrity="sha256-HtsXJanqjKTc8vVQjO4YMhiqFoXkfBsjBWcX91T1jr8=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/alien-16x16.png"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://apotiron.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Aline </span>Potiron</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications &amp; talks<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications &amp; talks</h1> <p class="post-description">Publications by categories in reversed chronological order.</p> </header> <article> <h2 id="peer-reviewed-articles">peer-reviewed articles</h2> <div class="publications"> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Kriaa2019" class="col-sm-8"> <div class="title">Microbial impact on cholesterol and bile acid metabolism: current status and future prospects</div> <div class="author"> <a href="https://fr.linkedin.com/in/aicha-kriaa-651679169" rel="external nofollow noopener" target="_blank">A. Kriaa</a>, M. Bourgin, <em>A. Potiron</em>, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'H. Mkaouar, A. Jablaoui, P. Gérard, E. Maguin, M. Rhimi' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Journal of Lipid Research</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1194/jlr.R088989"></span> <span class="__dimensions_badge_embed__" data-doi="10.1194/jlr.R088989" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Recently, the gut microbiota has emerged as a crucial factor that influences cholesterol metabolism. Ever since, significant interest has been shown in investigating these host-microbiome interactions to uncover microbiome-mediated functions on cholesterol and bile acid (BA) metabolism. Indeed, changes in gut microbiota composition and, hence, its derived metabolites have been previously reported to subsequently impact the metabolic processes and have been linked to several diseases. In this context, associations between a disrupted gut microbiome, impaired BA metabolism, and cholesterol dysregulation have been highlighted. Extensive advances in metagenomic and metabolomic studies in this field have allowed us to further our understanding of the role of intestinal bacteria in metabolic health and disease. However, only a few have provided mechanistic insights into their impact on cholesterol metabolism. Identifying the myriad functions and interactions of these bacteria to maintain cholesterol homeostasis remain an important challenge in such a field of research. In this review, we discuss the impact of gut microbiota on cholesterol metabolism, its association with disease settings, and the potential of modulating gut microbiota as a promising therapeutic target to lower hypercholesterolemia.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Kriaa2019</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Microbial impact on cholesterol and bile acid metabolism: current status and future prospects}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kriaa, A. and Bourgin, M. and Potiron, A. and Mkaouar, H. and Jablaoui, A. and Gérard, P. and Maguin, E. and Rhimi, M.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Lipid Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{60}</span><span class="p">,</span>
  <span class="na">issue</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{323--332}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{The American Society for Biochemistry and Molecular Biology}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1194/jlr.R088989}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{15397262}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Cholesterol metabolism,Gut microbiota,Hypercholesterolemia,Metabolic diseases}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Potiron2015" class="col-sm-8"> <div class="title">Recent Patents on Hypocholesterolemic Therapeutic Strategies: An Update</div> <div class="author"> <em>A. Potiron</em>, G. Philippe, T. Le Roy, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'P. Lesnik, E. Maguin, M. Rhimi' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Recent Advances in DNA &amp; Gene Sequences (Discontinued)</em>, 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.2174/2352092210666151216143459"></span> <span class="__dimensions_badge_embed__" data-doi="10.2174/2352092210666151216143459" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Worldwide, the cardiovascular diseases constitute a major cause of death with an ever growing incidence. Many medical approaches were developed against this physiopathology and patented; however up to now, no efficient treatment exists. Future developments are not only focusing on the identification of new therapeutic strategies against the cardiovascular diseases but also on a better understanding of the determinants of these multifactorial diseases. In this report, we reviewed the most recent patents that have been reported in this field of research.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Potiron2015</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Recent Patents on Hypocholesterolemic Therapeutic Strategies: An Update}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Potiron, A. and Philippe, G. and Le Roy, T. and Lesnik, P. and Maguin, E. and Rhimi, M.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Recent Advances in DNA &amp; Gene Sequences (Discontinued)}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{36--44}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.2174/2352092210666151216143459}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://www.eurekaselect.com/article/72538}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2352-0922/2352-0930}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Cardiovascular diseases,Atherosclerosis,Cholesterolemia,Therapeutic strategies,Patents, Healyh}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> </div> <h2 id="talks">talks</h2> <div class="publications"> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="PotironJuly2023" class="col-sm-8"> <div class="title">Towards an integrative view of explanation in human gut microbiome research</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>ISHPSSB Biennial Meeting 2023</em>, Jul 2023 </div> <div class="periodical"> University of Toronto, Canada </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/ISH23.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Experimental studies associate changing microbiomes with various healthy and diseased human phenotypes. Microbiomes are now integrated into the One Health concept, which considers the interconnections of animals, plants, humans and their shared environment to achieve better health outcomes (Berg et al., 2020). Clear explanations are rendered difficult by such complex and multilevel systems, initiating philosophical discussions about explanations, especially causal explanations. Indeed, Lynch and colleagues use a key explanatory standard inherited from 1880s microbiology, the Koch postulates, and a standard to establish causation in experimental science, the interventionist account, to assess the strength of causal claims about the microbiomes (Lynch et al., 2019). This approach is legitimate but overlooks part of microbiology’s history leading to an incomplete analysis of the assumptions used in this discipline. I propose here an alternative account of explanation in human gut microbiome research that considers this multipath history by recognizing the existence of an important alternative: the mechanistic account of explanation. First, I show that microbiology is not only a medical discipline. Parallel to the studies of the pathologist Koch, microbial ecology founders, Sergei Winogradsky and Martinus Beijerinck, studied microorganisms to learn about their role and behavior in their environment. Contemporary research in human gut microbiomes is considered part of both disciplines (Berg et al., 2020). Second, I argue that explanation in microbial ecology adopts a mechanistic-like strategy, which is not based on Koch’s postulates, but where the activity of the microbial community is explained by the interactions of the components of that community. For example, Winogradsky described how two groups of microorganisms interact with available resources and other microorganisms to explain the natural phenomenon of nitrification in the soil (transformation of ammonium to nitrate) (Winogradsky, 1949, 269). Finally, I develop a more comprehensive and encompassing account of explanation in human gut microbiome research. I propose to apply Craver’s mechanistic account to this discipline as he aims to understand constitutive explanation or “how the behavior of a whole is explained in terms of the behavior of its parts” (Craver, 2007, 160). The advantage of this account is that it can bind both explanatory traditions of microbiology by integrating the interventionist account in the selection of relevant causes. I argue that such an account is needed to accommodate contemporary appeal to a more holistic view (Berg et al., 2020), such as the One Health concept.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="PotironMar2023" class="col-sm-8"> <div class="title">How does the history of microbiology call for a more integrative view of explanation in human gut microbiome research?</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>Integrated History and Philosophy of Science - &amp;HPS9</em>, Mar 2023 </div> <div class="periodical"> University of South Carolina, Columbia, US </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/&amp;HPS923.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Human gut microbiome research is a growing field in microbiology and in philosophy. Experimental studies have associated microbiomes with various healthy and diseased human phenotypes, but use various definitions of “microbiome”. A collective recently attempted to define the microbiome as the microbiota – the microorganisms – and their “theatre of activity”, all their activities and metabolites (Berg et al., 2020). Clear explanations are rendered difficult by such complex and multilevel systems. Scientific claims about causal microbiomes have initiated philosophical discussions about explanations and in particular causal explanations in this domain. So far, the main study assessing this issue is the one developed by Lynch and colleagues (Lynch et al., 2019). They use a key explanatory standard inherited from 1880s microbiology, the Koch’s postulates, and a standard to establish causation in experimental science, the interventionist account to assess the strength of causal claims about the microbiomes. They conclude that few, if any, of these claims are causal. Following these discussions, this work analyzes the history of microbiology to suggest an account of scientific explanation in the context of human gut microbiome research. I build my argument on the realization that there is not a single understanding of explanation in microbiology. Focusing on Koch’s postulates and interventionism is legitimate but overlooks the existence of an important alternative: the mechanistic account of explanation. First, I retain the appeal to history of microbiology used by Lynch et al. to analyze what is an explanation in microbiology, but show that microbiology is not only a medical discipline. Indeed, the first microscopic observations and descriptions of microorganisms were done in an ecological mindset where the aim was to describe how these ‘animalcules’ live in their surrounding environments (Kolter, 2021). More significantly, parallel to Robert Koch’s development of his postulates (first enunciation in 1878, revisions in 1882), other microbiologists such as Sergei Winogradsky and Martinus Beijerinck studied microorganisms with the aim of knowing their role and behavior in their environment (Caumette et al., 2015). Both scientists are considered today as the founders of microbial ecology. Communications between these two sub-disciplines were not very intense (Kolter, 2021) but contemporary research in human gut microbiomes is considered as part of both disciplines (Berg et al., 2020; Caumette et al., 2015; Kolter, 2021). Therefore, second, I analyze how microorganisms alone or in community explain phenomena in microbial ecology. I take this analysis to be complementary to the analysis of explanation in medical microbiology. I claim that explanation in microbial ecology is not based on Koch’s postulates. First, microorganisms are not viewed only as pathogens. On the contrary, they are viewed as a part of the process of life: integrated in their environment and interacting with it. The notion of interaction is crucial in the understanding of (microbial) ecology. Second, ecological approaches look for biochemical pathways or mechanisms that explain some outcomes of environments colonized by microorganisms. For example, Winogradsky worked on the transformation of mineral compounds. In 1889, he discovered that two groups of microorganisms are involved in the nitrification cycle in the soil (transformation of ammonium to nitrate) and was interested in the discovery of chemical mechanisms of life process (Caumette et al., 2015,15-16). This call for a more mechanistic view of explanation in microbiology, where the activity of the microbial community is explained by the interactions of the components of that community. More contemporarily, Berg et al. (2020) advocate a holistic view of the human microbiome, not as separate from the host but as a whole with the host. Therefore, in a third claim, I argue that we need a more comprehensive and encompassing account of explanation in this discipline that considers this multipath history. It was also suggested by some replies to Lynch et al. (Klassen, 2020; Schneider, 2020). Here I do not offer such a detailed account but I suggest the possibility of using the mechanistic account developed by Craver (2007). Craver’s aim is the understanding of constitutive explanation or “how the behavior of a whole is explained in terms of the behavior of its parts” (Craver, 2007, 160). The advantage of this mechanistic account is that it can bind both explanatory traditions of microbiology by integrating the interventionist account in the selection of relevant causes. Such account has a normative target of distinguishing good from bad explanation in human gut microbiome research that can help scientists to make their claims clearer. Moreover, to overlook part of a discipline’s history when analyzing the assumptions used in this discipline is, philosophically speaking, problematic. A first step would be to apply Craver’s mechanistic account to human gut microbiome research. This would require secondly, to develop it forward by deepening our understanding of interlevel relationships and the difference between causal and constitutive explanations.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Potiron2022" class="col-sm-8"> <div class="title">Not everything is data: A revision of the relational framework of data</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>Sixth European Advanced School in the Philosophy of the Life Sciences - EASPLS</em>, Sep 2022 </div> <div class="periodical"> Bordeaux, France </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/Poster-EASPLS_v3.pdf" class="btn btn-sm z-depth-0" role="button">Poster</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The concept of data is related to the original dichotomy between observations of the world and theories about it. In the early 20 th century, the logical empiricists looked for a legitimization of the authority of scientific claims by appeal to the objectivity of observational reports or experiential data. In the late 1980’s philosophers and sociologists of sciences emphasized the importance of experimentation in the production of these data and therefore in the fact that data might be less objective than expected because of the human and instrumental manipulations needed to obtain them. Data remain representations of the world but a tension exists between their alleged objectivity and the fact that they are human made. Recently, a new framework was introduced, which defines the data according to their role within the scientific inquiry. This relational framework developed by Sabina Leonelli in her book Data centric biology: a philosophical study accounts very well for the contemporary uses of data such as big data and other data centric practices It defines data as any products of the scientific inquiry that have 1) the potential to travel between different situations of scientific inquiry and 2) the potential to be used as evidence to sustain knowledge claims. The aim of this work is twofold First, using a case study from data centric microbiology, viz. amplicon sequencing, I pinpoint some shortcomings of this frame work. Second, I propose a refinement of this framework to meet this challenge. Amplicon Sequencing (AS) is widely used in microbiology and microbial ecology to address the issue of the complexity of a microbial community in a given environment. The principle is the DNA sequencing and analysis of a single gene AS is expected to uncover the underlying characteristics of the microbial community in general. It is a methodological substitution to microbial observation which disrupts the link between microorganisms and their DNA. AS is a data centered method that generates millions of DNA sequences Therefore, it should, in principle, support the relational framework of data. First, I show that all the products of amplicon sequencing inquiry meet the two requi rements of the relational framework. However, scientists do distinguish among these products, which means, at least for practitioners of practice oriented philosophy of science that there is a prima facie problem I argue that there is continuity between these products throughout their production However, they have different roles in the scientific inquiry. Second, I propose refinements of the framework in order to respond to this problem. First, I emphasize the realization of a potentiality rather than the potentiality. Second, I focus on how the products of research activity are reused and for what purpose, rather than on their ability to travel. These working revisions allow me to identify by their role in the scientific inquiry, ’samples’ and a sub category of data that scientists called ’raw’. Because of the ambiguity of this term, I will use ’primary’ data</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="PotironJuly2021" class="col-sm-8"> <div class="title">How can highly contextual data such as metabarcoding data relate to the world?</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>ISHPSSB Biennial Meeting 2021</em>, Jul 2021 </div> <div class="periodical"> Cold Spring Harbor Laboratory, Online </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/ish21.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>This paper aims to reconsider the relationship between scientific data and the world. Data are usually considered as direct “screenshot” of the world. They are thus supposed to be fixed, context-independent and objective. In particular, in the context of modern Big Data-based science, data are supposed to “speak for themselves”. Metabarcoding is one method of modern microbiology that generates such data. Its principle is the DNA sequencing and analysis of a single gene, called a barcode. DNA sequences are thus central in those methods and passed from the world (the microorganisms) to the computer. During the transformation, choices are made which impaired the view of the fixed and objective screen shot. A sequenced barcode uniquely identifies the species of microorganisms in the community of a given environment. Thus, metabarcoding data are highly contextual where “context” is supposed to refer to a particular place, time and/or conditions. Thus, in metabarcoding, the path between data to knowledge is less straightforward than expected. This paper uses metabarcoding as a case study to develop a new understanding of the relationship between scientific data and the world. Metabarcoding generates millions of sequences, and is considered a data-centered method. Hence, I use the data journey framework developed by Leonelli in her book Data-centric biology: a philosophical study (2016). First, I assess what constitutes data in metabarcoding. I argue that DNA sequences become data not by acquiring the ability to travel but by acquiring the ability to sustain empirical claims. Sequences do so by increasing their reliability, through the interactions between environmental DNA and scientific technologies and through bioinformatics steps performed on sequences retrieved from sequencing platforms. Second, I evaluate how metabarcoding data can sustain empirical claims. I argue that the biological relevance of these data and conclusions made from them are better understood if we conceive the environment at the same epistemic level as the organism. Biology is usually conceived as gathering data and conclusions about individual organisms or even population. Yet, metabarcoding is about gathering data about a given environment. Projects such as the Earth Microbiome Project, in developing an ontology of studied environments that is more or less inclusive depending on the level of the taxonomy tend to integrate this shift.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="PotironOct2021" class="col-sm-8"> <div class="title">Human gut microbiomes research: what are the benefits of the mechanistic causal account?</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>The Philosophy Exchange Philosophy of Science Graduate Conference PSGC2021: From Theory to Practice</em>, Oct 2021 </div> <div class="periodical"> The London School of Economincs, Online </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/psgc21.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>This paper develops this working hypothesis: to focus on the interventionist framework of causal explanation in human gut microbiomes research underestimates the role of mechanistic account of causal explanation and it risks overlooking relevant therapeutics avenues. Human gut microbiomes studies are a recent and growing research field both in microbiology and in philosophy. Recently, causal explanation in this field have been studied using the interventionist framework of causal explanation. Lynch and colleagues advocate a methodological reductionism in order to achieve stronger causal explanation by improving the specificity, stability and proportionality of the causal relationship (Lynch, Parke, &amp; O’Malley, 2019). Attah and colleagues agreed but indicated that in some cases a more coarse-grained, less specified explanation is preferable (Attah, DiMarco, &amp; Plutynski, 2020). However, I argue that in human gut microbiomes studies, mechanistic causal accounts are used to strengthen causal relationship. Looking at the literature on the human gut microbiomes research linked with health outcomes, the published work can be divided into two camps. The first camp investigates single or few microorganisms (often bacteria) isolated from human microbiomes to study their effects both in vitro and in vivo. Researchers there aim to produce a probiotic: a bacterial strain with interesting properties for human health. I argue that despite relying on intervention such as giving the microorganism to mice in order to improve the phenotype of a disease, the causal relationship between the microorganism and the phenotype is strengthen when a mechanism is known or at least hypothesized (e.g., via the decrease of the gut barrier permeability). The second camp explores the role of whole microbiome in healthy and diseased phenotypes. There, the composition of the microbiome is uncovered mainly to be able to infer functions, when functions are not studied first. In her Philosophy of Microbiology (2014), O’Malley argued that functions here are understood in the ‘Cummins causal-role sense’ (p. 147). In this view, the functions of the components contribute to the function of the system containing them. This account can be labelled as mechanistic where functions from components (e.g., bacteria) are transmitted to the function of the system (e.g., whole microbiome). Both camps seem to value mechanistic causation in their explanation of phenotypes. I argue that there are at least two advantages of this account in comparison to the interventionist one. First, steps of the mechanism can be therapeutically targeted. Second, as the second camps analysis seem to show, researchers find relevant to study the whole microbiome and even consider it as causal. Mechanistic account does not suffer from the problem of specificity and proportionality and thus, as long as a mechanism between the whole microbiome and its effect is found, the whole microbiome can be causal. Focusing on the mechanistic account and not only the interventionist account makes more sense of the current state of therapeutic approaches and recent researches in the human gut microbiomes. This paper is a scientific case-study analysed with a philosophical perspective in order to challenge and complete the recent account of causality in microbiomes research.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="PotironNov2021" class="col-sm-8"> <div class="title">Challenging the philosophical definition of data using amplicon sequencing</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>Molecular Ecology and Evolution Seminar Autumn/Winter 2021</em>, Nov 2021 </div> <div class="periodical"> Bangor University, Online </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Potiron2017" class="col-sm-8"> <div class="title">Cholesterol conversion into coprostanol by bacterial strains isolated from the human gut microbiota can modify host cholesterol metabolism</div> <div class="author"> <em>A. Potiron</em>, <a href="https://fr.linkedin.com/in/aicha-kriaa-651679169" rel="external nofollow noopener" target="_blank">A. Kriaa</a>, S. Boudebbouze, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'A. Bruneau, M. Lhomme, P. Lesnik, E. Maguin, M. Rhimi, P. Gérard' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>7th Congress of European Microbiologists - FEMS</em>, Jul 2017 </div> <div class="periodical"> Valencia, Spain </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li></ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Potironetal2015" class="col-sm-8"> <div class="title">New insight in cholesterol metabolism by human gut microbiota</div> <div class="author"> <em>A. Potiron</em>, <a href="https://fr.linkedin.com/in/aicha-kriaa-651679169" rel="external nofollow noopener" target="_blank">A. Kriaa</a>, T. Le Roy, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'H. Mkaouar, S. Boudebbouze, P. Lesnik, P. Gérard, E. Maguin, M. Rhimi' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>26th Forum of the Tunisian Association of Biological Sciences</em>, Mar 2015 </div> <div class="periodical"> Monastir, Tunisia </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li></ol> </div> <h2 id="thesis">thesis</h2> <div class="publications"> <h2 class="bibliography">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="potiron:tel-01812929" class="col-sm-8"> <div class="title">Cholesterol conversion into coprostanol by bacteria from human gut microbiota and its impact cholesterolemia</div> <div class="author"> <em>Aline Potiron</em> </div> <div class="periodical"> Dec 2017 </div> <div class="periodical"> </div> <div class="links"> <a href="https://pastel.hal.science/tel-01812929/file/64723_POTIRON_2017_archivage.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Aline Potiron. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: August 17, 2023. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>