<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications &amp; talks | Aline Potiron</title> <meta name="author" content="Aline Potiron"> <meta name="description" content="Publications by categories (articles, talks and thesis) and in reversed chronological order."> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.4.0/css/all.min.css" integrity="sha256-HtsXJanqjKTc8vVQjO4YMhiqFoXkfBsjBWcX91T1jr8=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/alien-16x16.png"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://apotiron.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Aline </span>Potiron</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications &amp; talks<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications &amp; talks</h1> <p class="post-description">Publications by categories (articles, talks and thesis) and in reversed chronological order.</p> </header> <article> <h2>peer-reviewed articles</h2> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#031a40"><a href="https://link.springer.com/journal/13194" rel="external nofollow noopener" target="_blank">EJPS</a></abbr></div> <div id="Potiron2025" class="col-sm-8"> <div class="title">Beyond the microscope: Rethinking microbial diversity measurement with the model-based account</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>European Journal for Philosophy of Science</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s13194-025-00660-6"></span> <span class="__dimensions_badge_embed__" data-doi="10.1007/s13194-025-00660-6" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Measuring diversity in microbial ecology and microbiome studies is fraught with challenges, rendering the assessment of its “real-world” value nearly impossible. The instability of taxonomic classification, difficulty in isolating individuals, and reliance on DNA-based methods and statistical tools all contribute to the complexity of measuring diversity reliably. This manuscript explores the underlying philosophical issues, relating them to the measurement problem in philosophy. I argue that traditional philosophical accounts of measurement, including representational, operationalist, and realist approaches, are insufficient to address these issues. Instead, I examine these challenges through the lens of a model-based perspective on measurement, which can remain agnostic about entities and property ontologies, clarify the role of assumptions in diversity measurement, and provide solutions for justifying measurement procedures. This work emphasizes the importance of calibration and clearly defining measurement purposes, providing avenues for scientists to improve their measurement procedures. Ultimately, I contribute to a deeper understanding of the challenges and opportunities in measuring microbial diversity by bridging the gap between philosophy and scientific practice.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Potiron2025</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Beyond the microscope: Rethinking microbial diversity measurement with the model-based account}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Potiron, A.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{European Journal for Philosophy of Science}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Nature}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s13194-025-00660-6}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Amplicon sequencing, Diversity, Measurement theories, Microbial ecology, Microbiome, Philosophy in practice}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#18388b"><a href="https://link.springer.com/journal/10539" rel="external nofollow noopener" target="_blank">Biol Philos</a></abbr></div> <div id="Potiron2024" class="col-sm-8"> <div class="title">Different kinds of data: samples and the relational framework</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>Biology and Philosophy</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s10539-024-09962-0"></span> <span class="__dimensions_badge_embed__" data-doi="10.1007/s10539-024-09962-0" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>This paper proposes an original definition of samples as a kind of data within the relational framework of data. The distinction between scientific objects (e.g., samples, data, models) often needs to be clarified in the philosophy of science to understand their role in the scientific inquiry. The relational framework places data at the forefront of knowledge construction. Their epistemic status depends on their evaluation as potential evidence in a research situation and their ability to circulate among researchers. While samples are significant in data-generating science, their role has been underexplored in the philosophy of data literature. I draw on a case study from data-centric microbiology, viz. amplicon sequencing, to introduce specifications of the relational framework. These specifications capture the distinctive epistemic role of samples, allowing the discussion of their significance in the inquiry process. I argue that samples are necessarily transformed to be considered as evidence, portable in the limits of a situation, and they act as world anchors for claims about a phenomenon. I compare these specifications with other data and evidence frameworks and suggest they are compatible. The paper concludes by considering the extension of these criteria in the context of biobanking. The specifications proposed here help analyze other life sciences cases and deepen our understanding of samples and their epistemological role in scientific research.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Potiron2024</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Different kinds of data: samples and the relational framework}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Potiron, A.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Biology and Philosophy}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{39}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Nature}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10539-024-09962-0}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Samples, Data, Relational framework, Amplicon sequencing}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#7b6368"><a href="https://www.jlr.org/" rel="external nofollow noopener" target="_blank">JLR</a></abbr></div> <div id="Kriaa2019" class="col-sm-8"> <div class="title">Microbial impact on cholesterol and bile acid metabolism: current status and future prospects</div> <div class="author"> <a href="https://fr.linkedin.com/in/aicha-kriaa-651679169" rel="external nofollow noopener" target="_blank">A. Kriaa</a>, M. Bourgin, <em>A. Potiron</em>, <a href="https://tn.linkedin.com/in/h%C3%A9la-mkaouar-910b3380" rel="external nofollow noopener" target="_blank">H. Mkaouar</a>, A. Jablaoui, <a href="https://fr.linkedin.com/in/philippe-g%C3%A9rard-5784b032" rel="external nofollow noopener" target="_blank">P. Gérard</a>, <a href="https://fr.linkedin.com/in/emmanuelle-maguin-6b65922a" rel="external nofollow noopener" target="_blank">E. Maguin</a>, and <a href="https://fr.linkedin.com/in/moez-rhimi-528a875a" rel="external nofollow noopener" target="_blank">M. Rhimi</a> </div> <div class="periodical"> <em>Journal of Lipid Research</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1194/jlr.R088989"></span> <span class="__dimensions_badge_embed__" data-doi="10.1194/jlr.R088989" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Recently, the gut microbiota has emerged as a crucial factor that influences cholesterol metabolism. Ever since, significant interest has been shown in investigating these host-microbiome interactions to uncover microbiome-mediated functions on cholesterol and bile acid (BA) metabolism. Indeed, changes in gut microbiota composition and, hence, its derived metabolites have been previously reported to subsequently impact the metabolic processes and have been linked to several diseases. In this context, associations between a disrupted gut microbiome, impaired BA metabolism, and cholesterol dysregulation have been highlighted. Extensive advances in metagenomic and metabolomic studies in this field have allowed us to further our understanding of the role of intestinal bacteria in metabolic health and disease. However, only a few have provided mechanistic insights into their impact on cholesterol metabolism. Identifying the myriad functions and interactions of these bacteria to maintain cholesterol homeostasis remain an important challenge in such a field of research. In this review, we discuss the impact of gut microbiota on cholesterol metabolism, its association with disease settings, and the potential of modulating gut microbiota as a promising therapeutic target to lower hypercholesterolemia.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Kriaa2019</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Microbial impact on cholesterol and bile acid metabolism: current status and future prospects}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kriaa, A. and Bourgin, M. and Potiron, A. and Mkaouar, H. and Jablaoui, A. and Gérard, P. and Maguin, E. and Rhimi, M.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Lipid Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{60}</span><span class="p">,</span>
  <span class="na">issue</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{323--332}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{The American Society for Biochemistry and Molecular Biology}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1194/jlr.R088989}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{15397262}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Cholesterol metabolism,Gut microbiota,Hypercholesterolemia,Metabolic diseases}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Potiron2015" class="col-sm-8"> <div class="title">Recent Patents on Hypocholesterolemic Therapeutic Strategies: An Update</div> <div class="author"> <em>A. Potiron</em>, <a href="https://fr.linkedin.com/in/philippe-g%C3%A9rard-5784b032" rel="external nofollow noopener" target="_blank">P. Gérard</a>, T. Le Roy, P. Lesnik, <a href="https://fr.linkedin.com/in/emmanuelle-maguin-6b65922a" rel="external nofollow noopener" target="_blank">E. Maguin</a>, and <a href="https://fr.linkedin.com/in/moez-rhimi-528a875a" rel="external nofollow noopener" target="_blank">M. Rhimi</a> </div> <div class="periodical"> <em>Recent Advances in DNA &amp; Gene Sequences (Discontinued)</em>, 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.2174/2352092210666151216143459"></span> <span class="__dimensions_badge_embed__" data-doi="10.2174/2352092210666151216143459" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Worldwide, the cardiovascular diseases constitute a major cause of death with an ever growing incidence. Many medical approaches were developed against this physiopathology and patented; however up to now, no efficient treatment exists. Future developments are not only focusing on the identification of new therapeutic strategies against the cardiovascular diseases but also on a better understanding of the determinants of these multifactorial diseases. In this report, we reviewed the most recent patents that have been reported in this field of research.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Potiron2015</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Recent Patents on Hypocholesterolemic Therapeutic Strategies: An Update}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Potiron, A. and Gérard, P. and Le Roy, T. and Lesnik, P. and Maguin, E. and Rhimi, M.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Recent Advances in DNA &amp; Gene Sequences (Discontinued)}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{36--44}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.2174/2352092210666151216143459}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://www.eurekaselect.com/article/72538}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2352-0922/2352-0930}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Cardiovascular diseases,Atherosclerosis,Cholesterolemia,Therapeutic strategies,Patents, Healyh}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> </div> <h2>talks</h2> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#039"><a href="https://philsci.eu/" rel="external nofollow noopener" target="_blank">EPSA</a></abbr></div> <div id="PotironAugust2025b" class="col-sm-8"> <div class="title">Unearthing the ecological roots of microbiome studies</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>EPSA Biennial Meeting 2025</em>, Aug 2025 </div> <div class="periodical"> University of Groningen - Groningen, The Netherlands </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/EPSA-AP.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Animals host microorganisms at various body sites like the gastrointestinal tract, skin, lungs, vagina, and urinary tract. The human gut microbiome has been associated with multiple health and disease phenotypes for the past twenty years. Defining the “microbiome” is difficult, but a recent attempt at consensus has been published. The microbiome is the microbiota — the living microorganisms and their “theatre of activity” — all the molecules produced by the microorganisms in a biome understood as a habitat with distinct biological, physical, and chemical characteristics (Berg et al. 2020). I propose a new analysis of contemporary microbiome studies through the history of microbiology at the beginning of the 20th century that explains the duality of perspectives found in these studies. The development of microbiology follows two paths. A medical path and an ecological path. The medical one has received much of the historical, philosophical, and scientific attention through the study of the work of Robert Koch (among others). This path is dominated by a medical perspective characterized by methodological reductionism and linearity of causes. It is incarnated by developing the germ theory of diseases and using Koch’s postulates to find causes. It has been linked to contemporary trends in microbiome studies. For example, the research of causes in health-associated microbiome studies is mainly based on the original or variants of Koch’s postulates (e.g., Lynch et al. 2019; Neville et al. 2018). In this paper, I unearth the other path, characterized by an ecological perspective. I carefully analyze the work of the father of microbial ecology, Sergei Winogradsky (Winogradsky, 1949). For him, the first object of investigation is not the microorganism by itself but the processes happening in the soil. The challenge is to understand the complexity, the mess, of Nature. His later work, in particular, is distinctive from Koch’s approach. Among other things, he advocated a closer-to-reality experimental approach. Instead of using pure cultures in advantageous conditions, he strived for continuous experimentation on strains directly from the outside world and in conditions mimicking the natural ones. Winogradsky did not believe it would be possible to infer the behavior of the complex community of the soil from the behavior of the individual properties of isolated microorganisms. The reason is that the microorganisms are organized in the natural environment depending on the available resources. This environment will constrain the properties displayed by the microorganism. This interpretation is close to the contemporary understanding of emergent properties in microbial ecology and microbiome studies (Konopka, 2009; van den Berg et al., 2022). In sum, the ecological perspective is associated with the study of processes and flows of energy associated with a more complex view of causality and causes made of cycles, networks and the importance of the context for these relationships to happen. In this work, I explain the duality found in contemporary microbiome studies by the history of microbiology. I believe both perspectives are present in microbiome studies. The particular attention received by the medical path leaves the sensation that this duality is new or discovered in appeals to a more systematic approach. I argue that this duality is, in fact, rooted in the core of the discipline development. By connecting microbiome studies more profoundly to their own history, researchers can develop their perspectives and find old solutions to new problems. In particular, I criticize the medical perspective for its inadequacy to make sense of complex phenomena happening in the human gut microbiome. The ecological perspective can help better integrate more dynamic patterns of interactions between entities, understand and study emergent properties, and abandon causal linearity. Listening to the past for the microbiome studies means looking at managing the microorganismal community instead of eliminating targeted pathogens. Contemporary attempts at causal simplicity would be exceptions. More global approaches to health integrating physiology, nutrition, environment, etc., such as the one defended in the One Health concept, would have come sooner. </p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#24a74a"><a href="https://isme-microbes.org/" rel="external nofollow noopener" target="_blank">ISME</a></abbr></div> <div id="PotironAugust2025a" class="col-sm-8"> <div class="title">Unearthing the ecological roots of microbiome studies</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>ISME-Lat 2025</em>, Aug 2025 </div> <div class="periodical"> Merida, Mexico </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/ISME-LAT-AlinePotiron-final.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Microbiology historically developed from medicine and ecology. The medical path, based on germ theory and Koch’s postulates, is characterized by methodological reductionism and a linear understanding of cause and effect. However, this perspective is insufficient for understanding complex phenomena in the human gut microbiome. In contrast, Sergei Winogradsky’s ecological path embraces complexity and the “cycle of nature” concept. This path uses the elective method, which involves studying microorganisms under natural conditions and in communities. It recognizes that the natural environment constrains microorganisms’ properties and interactions and that causal relationships involve cycles, networks, and complex interactions. Contemporary microbiome studies often follow one path or the other but rarely both. In this talk, I argue that microbiology needs to acknowledge its ecological historical background to solve current challenges. Adopting an ecological perspective means shifting from targeting specific pathogens to managing the entire microorganismal community and studying dynamic interactions between biological entities and their environment. To achieve this, researchers can complement multiomics approaches with microbial cultures, shifting the focus from DNA and other molecules to the microorganisms, the properties of the communities they form, and their interactions with one another and with the environment. Computational approaches and simulations can also help investigate these interactions, alongside longitudinal studies that consider the spatiotemporal variability of the human gut microbiome seriously. Ultimately, interweaving conceptual and concrete solutions of both paths will lead to more integrative approaches to health, such as the one defended in the One Health concept.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:black"><a href="https://www.ishpssb.org/" rel="external nofollow noopener" target="_blank">ISHPSSB</a></abbr></div> <div id="PotironJuly2025ISH" class="col-sm-8"> <div class="title">Roundtable: Epistemic pluralism for a better microbiome research practice?</div> <div class="author"> D. Kenis, N. Morar, A. Nieves-Delgado, C. Pauvert, <em>A. Potiron</em>, and E. Schure</div> <div class="periodical"> <em>ISHPSSB Biennial Meeting 2025</em>, Jul 2025 </div> <div class="periodical"> University of Porto - Porto, Portugal </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Microbial ecological systems challenge fundamental biological notions like species and speciation which leads to uncertainties in measures of diversity, identification of functionality and the distinction between causality and correlation. The nature of microbial communities complicates sampling and preservation practices, as well as the design of successful experiments. While there is constant technological development, sequencing techniques are currently limited to the genetic level. Moreover, the different practices put into place to study the microbiome can have unwanted ethical consequences not only for humans, but also for animal models. In sum, the study of the human microbiome brings epistemic, methodological, technological and ethical challenges at different stages of the scientific inquiry. In this round table, we ask six experts in different fields (wet lab microbiology, philosophy of science, science and technology studies, integrated history and philosophy of science, and bioethics) to reflect on these problems from their disciplinary perspective. Through this interdisciplinary conversation we put to test what epistemic diversity and pluralism can offer to the study of complex systems like the microbiome. Can we abandon current limiting frameworks (e.g. genocentrism) and proxies (e.g. race) if we investigate the microbiome through a pluralistic perspective? Or are we left with a complexity difficult to operationalize? In the session, we prompt discussion with three questions which the speakers and the audience will discuss. These questions are: 1. What do we want to measure in the microbiome and why? What is measurable about the microbiome? 2. What is the most important challenge in the field of human microbiome research or in your practice? 3. Do you think interdisciplinary collaboration can help you overcome some of the challenges you face in your studies?</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="PotironJuly2025" class="col-sm-8"> <div class="title">Invited: Microbiome and Philosophy: Origins, Causality, and Human Categorizations</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>Microbiome Engineering - El Aidy Lab</em>, Jul 2025 </div> <div class="periodical"> University of Amsterdam - Amsterdam, The Netherlands </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/ElAidyLabTalk.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Microbiomes, especially gut microbiomes, have been associated with various diseases, from colorectal cancer to anxiety disorders. The relevance of microbiomes for health has risen to public, historical, and philosophical attention. Far from doing philosophy from afar, I adopt a “practice-oriented” approach to philosophy. That is, I analyze the scientific practices that produce and improve knowledge in microbiome studies and microbiology. In this talk, I focus on three aspects of this philosophical attention and their connections to scientific practices. First, I investigate the impact of the historical development of the field of microbiome on the different perspectives in contemporary microbiome studies. By unearthing the ecological root of the discipline, I argue that more collaboration and integration are needed to solve current challenges between what I call a medical and an ecological perspective on the microbiome. Second, I explore the issues surrounding the establishment of causal relationships. Starting from the fact that the discipline employs several methods to establish causal claims and seeks theoretical backing to ascertain and evaluate those claims, I provide an overview of the philosophical positions available to scientists to make sense of causal claims. Finally, focusing on a recent trend, primarily since 2018, in microbiome studies that utilize human categorizations to investigate human health disparities and inform personalized medicine, I argue that this approach is problematic and should be reconsidered. While potentially useful for making inferences and generalizations, I question the inferential power of these categories and argue that there are more risks than benefits associated with their use.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#046592"><a href="https://www.ozsw.nl/activity/philosophy-of-science-in-a-forest-2025/" rel="external nofollow noopener" target="_blank">PSF</a></abbr></div> <div id="PotironMay20252" class="col-sm-8"> <div class="title">Microbiome Research and the Myth of Ethno-racial Categories</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>Philosophy of Science in a Forest</em>, May 2025 </div> <div class="periodical"> Internationale School for Wijsbegeerte - Leusden, The Netherlands </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/PSF2025.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The increasing use of ethno-racial categories in microbiome studies has sparked debate among scholars, with concerns surrounding the problematic narrative of "Westernized" vs. "non-Westernized" that reifies racist views (for a review, see Rawson 2024). I focus on the more recent trend in microbiome studies, roughly after 2018, where ethno-racial categories are used to study human health disparities to inform personalized medicine. I argue that this use is problematic and should be abandoned. In this paper, I question the inferential power of ethno-racial categories in microbiome studies in the context of personalized medicine on three points. Firstly, in the context of medical genetics, it has been argued that the definitions of ethno-racial categories are ambiguous and inconsistent, creating epistemic uncertainty (Malinowska and Serpico, 2023). I argue that this conceptual imprecision also plagues microbiome studies. These studies often use interchangeably different concepts such as “race,” “ethnicity,” “geography,” and “nationality.” Moreover, few studies define these terms before using them or do not use them consistently throughout one paper. Additionally, when institutional or more general definitions backed up such papers, those definitions are not consistent globally and depend mainly on the location of the researchers (e.g., USA vs Netherlands). Ultimately, this situation prevents comparative analyses of different studies and hinders scientific communication. Secondly, these categories reduce multiple dimensions, including human genetic diversity, diet, and socio-cultural dimensions, into one category, which can result in stereotyping and essentializing ethno-racial categories through the microbiome. This unclarity of the actual variables of interest also creates confusion in cause-effect relationships: It is unclear whether the ethno-racial category is a cause of the microbiome composition or if it is an effect of the diet (or another dimension) mediated by the microbiome. Finally, few studies have found consistent, reliable associations between microbial taxa and disease or health phenotypes. Similarly, no taxa are reliably and consistently associated with a particular human allele. Researchers tiptoed on what is really relevant for a disease in the ethno-racial category (i.e., a genetic variant, a nutrient in the diet, etc.). However, these factors could lead to different classifications of the same human population, rendering ethno- racial categories useless for treating a particular individual. Overall, I argue that ethno-racial categories in microbiome studies have limited inferential power. The risks of using ethno-racial categories in microbiome studies, including poor scientific communication, attributing a spurious causal role to these categories, stereotyping, and misplacing individuals into categories that may not be relevant for treatment or risk-factor assessment, outweigh any potential benefits. Instead, researchers should focus on the actual variables they are interested in, such as diet, to avoid racist descriptions and the microbiomization of ethno-racial categories.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#4b766b"><a href="https://rotorub.wordpress.com/" rel="external nofollow noopener" target="_blank">ROTO</a></abbr></div> <div id="PotironMay2025" class="col-sm-8"> <div class="title">Invited: Microbiome foundations and faultlines through the lens of history and philosophy</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>ROTO Lecture Series</em>, May 2025 </div> <div class="periodical"> Ruhr-University Bochum, Online </div> <div class="links"> <a href="/assets/pdf/ROTOLectureSeries.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#af1f24"><a href="https://philsci.org/" rel="external nofollow noopener" target="_blank">PSA</a></abbr></div> <div id="PotironNovember2024" class="col-sm-8"> <div class="title">Diversity measurement in microbial ecology</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>PSA Biennial Meeting 2024</em>, Nov 2024 </div> <div class="periodical"> As part of the Symposium "Data Practices in Microbiology and Microbiome Science: Unpacking Upstream Choices in Experimental Design" with V. Keiser, D. Beiting, and J. Wallis - New Orleans, USA </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/PSA24_AlinePotiron.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Taxonomic diversity is a common measure in the scientific literature, yet a philosophical analysis of diversity as measurement remains elusive. Here, I propose such an analysis for the case of microbial ecology. I contend that upstream choices made by researchers before generating measurement data impact the value assigned to diversity. The talk outlines the implications of this analysis for the model-based accounts of measurement and the species concept. First, I analyze DNA-based microbial diversity assessment using model-based accounts of measurement. In these accounts, measurements are related to theoretical and statistical modeling. Measurements have a concrete and representational level (Tal 2013). When measuring a quantity, here diversity, of an object of interest, here microbial community, both need to be idealized before inferring the value for that quantity. For example, researchers can choose richness as an idealization of diversity and a molecular marker as an idealization of the object of interest, the community. An ideal solution is then sought, assumed to be attainable by ideal measuring instruments, i.e., instruments that give traceable results to appropriate standards. Researchers can then choose which instruments to use – the DNA sequencers, the bioinformatics tools, etc. Finally, de-idealization is required to obtain the measured value. Here, the concept of species comes in. The ideal measurement entails that there is an ideal way of distinguishing species. Still, in the actual measurement, researchers make assumptions about this, e.g., similarity thresholds at which two DNA sequences of the same marker are from different species. The measure’s validity depends not only on the empirical qualities of the measurement but also on the adequacy of the idealizations and the degree of correspondence between the ideal and the theoretical measurement process (Giordani and Mari 2012). Next, I assess the consequences for model-based accounts of measurement. Even if these accounts are suitable to analyze this case and highlight some of the choices needed in the production of the diversity measure, it is not easy to see how they can guide researchers in their choices. The choice of the model itself is not straightforward – for example, which instrument to include in the model or which standards to use. These choices will impact the value of the quantity researched – diversity – and what can be said about it. These accounts might need modifications. Finally, I assess the role of the species concept. This concept enters measurement in the modeling activities, thus creating relevant uncertainties to the diversity parameter—the diversity value changes depending on which concrete definition of species researchers choose. I explore if the need for a concrete definition is at odds with the argument for a general definition of species in the context of conservation biology, where diversity measures are compared.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#8e1631"><a href="https://wcprome2024.com/" rel="external nofollow noopener" target="_blank">WCP2024</a></abbr></div> <div id="PotironAugust2024" class="col-sm-8"> <div class="title">Practice-first approach: from biological cases to philosophical issues</div> <div class="author"> <em>A. Potiron</em>, S. Haas, M. Casali, and F. Rheinhardt</div> <div class="periodical"> <em>XXV World Congress of Philosophy</em>, Aug 2024 </div> <div class="periodical"> Rome, Italy </div> <div class="links"> <a href="/assets/pdf/WCP2024.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#52964c"><a href="https://www.jku.at/en/institute-of-philosophy-and-scientific-method/epicause-2024/" rel="external nofollow noopener" target="_blank">EpiCause2024</a></abbr></div> <div id="PotironMay2024" class="col-sm-8"> <div class="title">A new light on causes in human health-associated microbiome studies by unearthing its ecological roots</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>Causality in Epidemiology - CitS conference series</em>, May 2024 </div> <div class="periodical"> Linz, Austria </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/EpiCause2024.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Animals host microorganisms at various body sites like the gastrointestinal tract, skin, lungs, vagina, and urinary tract. The human gut microbiome has been associated with multiple health and disease phenotypes for the past twenty years. This tendency culminates with the integration of microbiomes in epidemiology studies. The research of causes in health-associated microbiome studies is based on the original or variants of Koch’s postulates (e.g., Lynch et al., 2019; Vonaesh et al., 2018). They are helpful for the detection of single and specific causal factors of diseases, e.g., microorganisms. They have been successful for some time, especially in the case of monocausal diseases, and gave rise to the germ theory of diseases. The ideas of specificity in terms of monocausality and homogeneity of causes are pervasive in contemporary medicine (Ross, 2018). They are also ubiquitous in microbiome studies, e.g., researching biomarkers for cancers or researching a curative “silver bullet” bacteria. As a result, the default assumptions in those researches are often a monocausal and specific cause for a disease or an intervention. However, I argue that scientists in health-associated microbiome studies have forgotten their history. Microbiology and microbiome studies are also ecological disciplines, not only medical ones. The father of microbial ecology is Sergei Winogradsky. While younger than Robert Koch, he was one of his contemporaries. In Winogradsky’s work, the research of causes is not focused on entities but on processes and interactions. He was studying the natural phenomena of the soil and the microorganismal community within it. He recognized at face values that those phenomena were complex by nature, i.e., multifactorial and non-entity specific. In this work, I will explore the state of health-associated microbiome studies if the default position would have been “Winogradskian,” i.e., looking for several factors and non-specific entities before looking for specific and unique causes. It might be then that the contemporary attempts at simplicity would be an exception rather than a rule and that more global and integrative approaches, such as the one defended in the OneHealth concept, would have come sooner, impacting epidemiology-related policies. </p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#039"><a href="https://philsci.eu/" rel="external nofollow noopener" target="_blank">EPSA</a></abbr></div> <div id="PotironSeptember2023" class="col-sm-8"> <div class="title">Species is a unit of measurement</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>EPSA Biennial Meeting 2023</em>, Sep 2023 </div> <div class="periodical"> Belgrade, Serbia </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/EPSA23.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Species is a concept that has long been studied by biologists and philosophers, giving rise to several practical, epistemological, and theoretical questions now known as the “species problem”. Species are considered units of classification and evolution, but also units of diversity. In microbial ecology, in particular, diversity can be assessed using DNA sequence data. Yet, this process requires material tools such as DNA sequencers as well as bioinformatic treatments, both of which influence the value of diversity. While species diversity is often associated with a measure in the literature, a philosophical analysis of diversity as measurement remains elusive. This paper proposes to use the species concept in the context of diversity-as-a-measurement and outlines its implications for the practical, epistemological, and theoretical dimensions of the species problem. First, I analyse the DNA-based assessment of microbial diversity using the ideas of model-based accounts of measurement. In these accounts, measurements are related to theoretical and/or statistical modelling. In particular, measurements have a concrete and a representational, abstract level (Tal, 2013). When measuring a quantity, here diversity, of an object of interest, here microbial community, both need to be idealised before inferring the value for that quantity. For example, richness is an idealisation of the quantity diversity, and a molecular marker is an idealisation of the object of interest – the community. An ideal solution is then sought, which is assumed to be attainable by ideal measuring instruments, i.e. instruments that give results that are traceable to appropriate standards. Following my example, the measuring instrument is a combination of DNA sequencers and bioinformatics tools. Finally, de-idealisation is required to obtain the measured value. This is where the concept of species comes in. Indeed, the ideal measurement entails that there is an ideal way of distinguishing species, but the actual measurement makes some assumptions about this, e.g., thresholds at which two DNA sequences of the same marker are from different species. The validity of the measure depends on the empirical qualities of the measurement, but also the adequacy of the idealisations and the degree of correspondence between the ideal and the theoretical measurement process (Giordani and Mari, 2012). In this analysis, the species concept is one of many sources of uncertainty in the diversity measured by a measurement process. Second, I assess the implications of this analysis for the practical dimensions of the species problems. In the practice of microbial ecology, the definition of species is not so important as long as the assumptions in the measurement process are clearly stated and as long as the degree of uncertainty reflects these assumptions. This also makes the measure highly contextual (Shade, 2016). Yet, in the broader context of conservation biology, where diversity measures are compared, the contrary has been argued, i.e., a general definition of species is used to measure diversity across environments (Amitani, 2022). It would therefore be interesting to examine the measurement process in this discipline, where and how the species concept plays a role in this process, and whether the conclusion about the importance of the problem still holds. Third, I explore the epistemic role of the species concept as a unit of measurement and its implications for the homonymy thesis. Species are a source of uncertainty about the measure of the quantity diversity. On the other hand, it has been argued that there is an epistemological disunity between three other epistemic roles of the species concept: classification, generalisation, and evolution (Reydon, 2022, 76). Thus, it remains to be examined whether this new epistemic use is associated with a concept of species that is coextensive with the ones used in the above-mentioned epistemological roles. Moreover, the disunity argument partly grounds the homonymy thesis, according to which the term ‘species’ has several meanings at the same time and these meanings refer to multiple and distinct entities in the world (Reydon, 2022, 74). Therefore, analysing the concept(s) of species used in uncertainty assessment in measurement processes of diversity can strengthen or not the homonymy thesis. Finally, I follow the practice-oriented philosophy project to investigate the theoretical assumptions and the explanatory importance of the species concept in diversity measurement. In the case of measuring the diversity of a microbial community using a molecular marker, DNA sequences encountered in a delineated community and spatial region are compared and then assigned to groups that we call species. The sequences are considered different enough between groups, and the changes in the nucleotide sequence are sufficiently large to qualify for the status of species. This is close to the theoretical idea of ‘evolutionary independence’. This idea has been developed to save the technical use of the term ‘species’ in the ‘species-as-status’ thesis. In this thesis, because of the acceptance of the homonymy thesis, the concept of species lacks conceptual unity and a real entity to which it can be applied, so it has no ontological meaning (Reydon, 2022). In addition, in my example, diversity is one parameter of community ecology that has to be explained (Shade, 2016). Thus, it would be interesting to deepen and diversify the analysis of measurement processes of diversity to understand the explanatory importance of the species concept in these contexts.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:black"><a href="https://www.ishpssb.org/" rel="external nofollow noopener" target="_blank">ISHPSSB</a></abbr></div> <div id="PotironJuly2023" class="col-sm-8"> <div class="title">Towards an integrative view of explanation in human gut microbiome research</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>ISHPSSB Biennial Meeting 2023</em>, Jul 2023 </div> <div class="periodical"> University of Toronto, Canada </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/ISH23.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Experimental studies associate changing microbiomes with various healthy and diseased human phenotypes. Microbiomes are now integrated into the One Health concept, which considers the interconnections of animals, plants, humans and their shared environment to achieve better health outcomes (Berg et al., 2020). Clear explanations are rendered difficult by such complex and multilevel systems, initiating philosophical discussions about explanations, especially causal explanations. Indeed, Lynch and colleagues use a key explanatory standard inherited from 1880s microbiology, the Koch postulates, and a standard to establish causation in experimental science, the interventionist account, to assess the strength of causal claims about the microbiomes (Lynch et al., 2019). This approach is legitimate but overlooks part of microbiology’s history leading to an incomplete analysis of the assumptions used in this discipline. I propose here an alternative account of explanation in human gut microbiome research that considers this multipath history by recognizing the existence of an important alternative: the mechanistic account of explanation. First, I show that microbiology is not only a medical discipline. Parallel to the studies of the pathologist Koch, microbial ecology founders, Sergei Winogradsky and Martinus Beijerinck, studied microorganisms to learn about their role and behavior in their environment. Contemporary research in human gut microbiomes is considered part of both disciplines (Berg et al., 2020). Second, I argue that explanation in microbial ecology adopts a mechanistic-like strategy, which is not based on Koch’s postulates, but where the activity of the microbial community is explained by the interactions of the components of that community. For example, Winogradsky described how two groups of microorganisms interact with available resources and other microorganisms to explain the natural phenomenon of nitrification in the soil (transformation of ammonium to nitrate) (Winogradsky, 1949, 269). Finally, I develop a more comprehensive and encompassing account of explanation in human gut microbiome research. I propose to apply Craver’s mechanistic account to this discipline as he aims to understand constitutive explanation or “how the behavior of a whole is explained in terms of the behavior of its parts” (Craver, 2007, 160). The advantage of this account is that it can bind both explanatory traditions of microbiology by integrating the interventionist account in the selection of relevant causes. I argue that such an account is needed to accommodate contemporary appeal to a more holistic view (Berg et al., 2020), such as the One Health concept.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge"><a href="http://integratedhps.org/" rel="external nofollow noopener" target="_blank">&amp;HPS</a></abbr></div> <div id="PotironMar2023" class="col-sm-8"> <div class="title">How does the history of microbiology call for a more integrative view of explanation in human gut microbiome research?</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>Integrated History and Philosophy of Science - &amp;HPS9</em>, Mar 2023 </div> <div class="periodical"> University of South Carolina, Columbia, USA </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/&amp;HPS923.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Human gut microbiome research is a growing field in microbiology and in philosophy. Experimental studies have associated microbiomes with various healthy and diseased human phenotypes, but use various definitions of “microbiome”. A collective recently attempted to define the microbiome as the microbiota – the microorganisms – and their “theatre of activity”, all their activities and metabolites (Berg et al., 2020). Clear explanations are rendered difficult by such complex and multilevel systems. Scientific claims about causal microbiomes have initiated philosophical discussions about explanations and in particular causal explanations in this domain. So far, the main study assessing this issue is the one developed by Lynch and colleagues (Lynch et al., 2019). They use a key explanatory standard inherited from 1880s microbiology, the Koch’s postulates, and a standard to establish causation in experimental science, the interventionist account to assess the strength of causal claims about the microbiomes. They conclude that few, if any, of these claims are causal. Following these discussions, this work analyzes the history of microbiology to suggest an account of scientific explanation in the context of human gut microbiome research. I build my argument on the realization that there is not a single understanding of explanation in microbiology. Focusing on Koch’s postulates and interventionism is legitimate but overlooks the existence of an important alternative: the mechanistic account of explanation. First, I retain the appeal to history of microbiology used by Lynch et al. to analyze what is an explanation in microbiology, but show that microbiology is not only a medical discipline. Indeed, the first microscopic observations and descriptions of microorganisms were done in an ecological mindset where the aim was to describe how these ‘animalcules’ live in their surrounding environments (Kolter, 2021). More significantly, parallel to Robert Koch’s development of his postulates (first enunciation in 1878, revisions in 1882), other microbiologists such as Sergei Winogradsky and Martinus Beijerinck studied microorganisms with the aim of knowing their role and behavior in their environment (Caumette et al., 2015). Both scientists are considered today as the founders of microbial ecology. Communications between these two sub-disciplines were not very intense (Kolter, 2021) but contemporary research in human gut microbiomes is considered as part of both disciplines (Berg et al., 2020; Caumette et al., 2015; Kolter, 2021). Therefore, second, I analyze how microorganisms alone or in community explain phenomena in microbial ecology. I take this analysis to be complementary to the analysis of explanation in medical microbiology. I claim that explanation in microbial ecology is not based on Koch’s postulates. First, microorganisms are not viewed only as pathogens. On the contrary, they are viewed as a part of the process of life: integrated in their environment and interacting with it. The notion of interaction is crucial in the understanding of (microbial) ecology. Second, ecological approaches look for biochemical pathways or mechanisms that explain some outcomes of environments colonized by microorganisms. For example, Winogradsky worked on the transformation of mineral compounds. In 1889, he discovered that two groups of microorganisms are involved in the nitrification cycle in the soil (transformation of ammonium to nitrate) and was interested in the discovery of chemical mechanisms of life process (Caumette et al., 2015,15-16). This call for a more mechanistic view of explanation in microbiology, where the activity of the microbial community is explained by the interactions of the components of that community. More contemporarily, Berg et al. (2020) advocate a holistic view of the human microbiome, not as separate from the host but as a whole with the host. Therefore, in a third claim, I argue that we need a more comprehensive and encompassing account of explanation in this discipline that considers this multipath history. It was also suggested by some replies to Lynch et al. (Klassen, 2020; Schneider, 2020). Here I do not offer such a detailed account but I suggest the possibility of using the mechanistic account developed by Craver (2007). Craver’s aim is the understanding of constitutive explanation or “how the behavior of a whole is explained in terms of the behavior of its parts” (Craver, 2007, 160). The advantage of this mechanistic account is that it can bind both explanatory traditions of microbiology by integrating the interventionist account in the selection of relevant causes. Such account has a normative target of distinguishing good from bad explanation in human gut microbiome research that can help scientists to make their claims clearer. Moreover, to overlook part of a discipline’s history when analyzing the assumptions used in this discipline is, philosophically speaking, problematic. A first step would be to apply Craver’s mechanistic account to human gut microbiome research. This would require secondly, to develop it forward by deepening our understanding of interlevel relationships and the difference between causal and constitutive explanations.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Potiron2022" class="col-sm-8"> <div class="title">Not everything is data: A revision of the relational framework of data</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>Sixth European Advanced School in the Philosophy of the Life Sciences - EASPLS</em>, Sep 2022 </div> <div class="periodical"> Bordeaux, France </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/Poster-EASPLS_v3.pdf" class="btn btn-sm z-depth-0" role="button">Poster</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The concept of data is related to the original dichotomy between observations of the world and theories about it. In the early 20 th century, the logical empiricists looked for a legitimization of the authority of scientific claims by appeal to the objectivity of observational reports or experiential data. In the late 1980’s philosophers and sociologists of sciences emphasized the importance of experimentation in the production of these data and therefore in the fact that data might be less objective than expected because of the human and instrumental manipulations needed to obtain them. Data remain representations of the world but a tension exists between their alleged objectivity and the fact that they are human made. Recently, a new framework was introduced, which defines the data according to their role within the scientific inquiry. This relational framework developed by Sabina Leonelli in her book Data centric biology: a philosophical study accounts very well for the contemporary uses of data such as big data and other data centric practices It defines data as any products of the scientific inquiry that have 1) the potential to travel between different situations of scientific inquiry and 2) the potential to be used as evidence to sustain knowledge claims. The aim of this work is twofold First, using a case study from data centric microbiology, viz. amplicon sequencing, I pinpoint some shortcomings of this frame work. Second, I propose a refinement of this framework to meet this challenge. Amplicon Sequencing (AS) is widely used in microbiology and microbial ecology to address the issue of the complexity of a microbial community in a given environment. The principle is the DNA sequencing and analysis of a single gene AS is expected to uncover the underlying characteristics of the microbial community in general. It is a methodological substitution to microbial observation which disrupts the link between microorganisms and their DNA. AS is a data centered method that generates millions of DNA sequences Therefore, it should, in principle, support the relational framework of data. First, I show that all the products of amplicon sequencing inquiry meet the two requi rements of the relational framework. However, scientists do distinguish among these products, which means, at least for practitioners of practice oriented philosophy of science that there is a prima facie problem I argue that there is continuity between these products throughout their production However, they have different roles in the scientific inquiry. Second, I propose refinements of the framework in order to respond to this problem. First, I emphasize the realization of a potentiality rather than the potentiality. Second, I focus on how the products of research activity are reused and for what purpose, rather than on their ability to travel. These working revisions allow me to identify by their role in the scientific inquiry, ’samples’ and a sub category of data that scientists called ’raw’. Because of the ambiguity of this term, I will use ’primary’ data</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:black"><a href="https://www.ishpssb.org/" rel="external nofollow noopener" target="_blank">ISHPSSB</a></abbr></div> <div id="PotironJuly2021" class="col-sm-8"> <div class="title">How can highly contextual data such as metabarcoding data relate to the world?</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>ISHPSSB Biennial Meeting 2021</em>, Jul 2021 </div> <div class="periodical"> Cold Spring Harbor Laboratory, Online </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/ish21.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>This paper aims to reconsider the relationship between scientific data and the world. Data are usually considered as direct “screenshot” of the world. They are thus supposed to be fixed, context-independent and objective. In particular, in the context of modern Big Data-based science, data are supposed to “speak for themselves”. Metabarcoding is one method of modern microbiology that generates such data. Its principle is the DNA sequencing and analysis of a single gene, called a barcode. DNA sequences are thus central in those methods and passed from the world (the microorganisms) to the computer. During the transformation, choices are made which impaired the view of the fixed and objective screen shot. A sequenced barcode uniquely identifies the species of microorganisms in the community of a given environment. Thus, metabarcoding data are highly contextual where “context” is supposed to refer to a particular place, time and/or conditions. Thus, in metabarcoding, the path between data to knowledge is less straightforward than expected. This paper uses metabarcoding as a case study to develop a new understanding of the relationship between scientific data and the world. Metabarcoding generates millions of sequences, and is considered a data-centered method. Hence, I use the data journey framework developed by Leonelli in her book Data-centric biology: a philosophical study (2016). First, I assess what constitutes data in metabarcoding. I argue that DNA sequences become data not by acquiring the ability to travel but by acquiring the ability to sustain empirical claims. Sequences do so by increasing their reliability, through the interactions between environmental DNA and scientific technologies and through bioinformatics steps performed on sequences retrieved from sequencing platforms. Second, I evaluate how metabarcoding data can sustain empirical claims. I argue that the biological relevance of these data and conclusions made from them are better understood if we conceive the environment at the same epistemic level as the organism. Biology is usually conceived as gathering data and conclusions about individual organisms or even population. Yet, metabarcoding is about gathering data about a given environment. Projects such as the Earth Microbiome Project, in developing an ontology of studied environments that is more or less inclusive depending on the level of the taxonomy tend to integrate this shift.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge"><a href="https://philosophyexchange.org/" rel="external nofollow noopener" target="_blank">PX</a></abbr></div> <div id="PotironOct2021" class="col-sm-8"> <div class="title">Human gut microbiomes research: what are the benefits of the mechanistic causal account?</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>The Philosophy Exchange Philosophy of Science Graduate Conference PSGC2021: From Theory to Practice</em>, Oct 2021 </div> <div class="periodical"> The London School of Economics, Online </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="/assets/pdf/psgc21.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>This paper develops this working hypothesis: to focus on the interventionist framework of causal explanation in human gut microbiomes research underestimates the role of mechanistic account of causal explanation and it risks overlooking relevant therapeutics avenues. Human gut microbiomes studies are a recent and growing research field both in microbiology and in philosophy. Recently, causal explanation in this field have been studied using the interventionist framework of causal explanation. Lynch and colleagues advocate a methodological reductionism in order to achieve stronger causal explanation by improving the specificity, stability and proportionality of the causal relationship (Lynch, Parke, &amp; O’Malley, 2019). Attah and colleagues agreed but indicated that in some cases a more coarse-grained, less specified explanation is preferable (Attah, DiMarco, &amp; Plutynski, 2020). However, I argue that in human gut microbiomes studies, mechanistic causal accounts are used to strengthen causal relationship. Looking at the literature on the human gut microbiomes research linked with health outcomes, the published work can be divided into two camps. The first camp investigates single or few microorganisms (often bacteria) isolated from human microbiomes to study their effects both in vitro and in vivo. Researchers there aim to produce a probiotic: a bacterial strain with interesting properties for human health. I argue that despite relying on intervention such as giving the microorganism to mice in order to improve the phenotype of a disease, the causal relationship between the microorganism and the phenotype is strengthen when a mechanism is known or at least hypothesized (e.g., via the decrease of the gut barrier permeability). The second camp explores the role of whole microbiome in healthy and diseased phenotypes. There, the composition of the microbiome is uncovered mainly to be able to infer functions, when functions are not studied first. In her Philosophy of Microbiology (2014), O’Malley argued that functions here are understood in the ‘Cummins causal-role sense’ (p. 147). In this view, the functions of the components contribute to the function of the system containing them. This account can be labelled as mechanistic where functions from components (e.g., bacteria) are transmitted to the function of the system (e.g., whole microbiome). Both camps seem to value mechanistic causation in their explanation of phenotypes. I argue that there are at least two advantages of this account in comparison to the interventionist one. First, steps of the mechanism can be therapeutically targeted. Second, as the second camps analysis seem to show, researchers find relevant to study the whole microbiome and even consider it as causal. Mechanistic account does not suffer from the problem of specificity and proportionality and thus, as long as a mechanism between the whole microbiome and its effect is found, the whole microbiome can be causal. Focusing on the mechanistic account and not only the interventionist account makes more sense of the current state of therapeutic approaches and recent researches in the human gut microbiomes. This paper is a scientific case-study analysed with a philosophical perspective in order to challenge and complete the recent account of causality in microbiomes research.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="PotironNov2021" class="col-sm-8"> <div class="title">Invited: Challenging the philosophical definition of data using amplicon sequencing</div> <div class="author"> <em>A. Potiron</em> </div> <div class="periodical"> <em>Molecular Ecology and Evolution Seminar Autumn/Winter 2021</em>, Nov 2021 </div> <div class="periodical"> Bangor University, Online </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#804d98"><a href="https://www.iata.csic.es/en" rel="external nofollow noopener" target="_blank">IATA</a></abbr></div> <div id="PotironJuly2017" class="col-sm-8"> <div class="title">Invited: How the conversion of cholesterol into coprostanol by intestinal bacteria can modulate host cholesterol metabolism</div> <div class="author"> <em>A. Potiron</em>, <a href="https://fr.linkedin.com/in/aicha-kriaa-651679169" rel="external nofollow noopener" target="_blank">A. Kriaa</a>, S. Boudebbouze, A. Bruneau, M. Lhomme, P. Lesnik, <a href="https://fr.linkedin.com/in/emmanuelle-maguin-6b65922a" rel="external nofollow noopener" target="_blank">E. Maguin</a>, <a href="https://fr.linkedin.com/in/moez-rhimi-528a875a" rel="external nofollow noopener" target="_blank">M. Rhimi</a>, and <a href="https://fr.linkedin.com/in/philippe-g%C3%A9rard-5784b032" rel="external nofollow noopener" target="_blank">P. Gérard</a> </div> <div class="periodical"> <em>Institute of Agrochemistry and Food Technology - IATA</em>, Jul 2017 </div> <div class="periodical"> Valencia, Spain </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Cardiovascular diseases are the leading cause of death in the world and affect more than 200million people. Cholesterol level management is a keystone to limit these diseases. Several treatments exist but patients complain about heavy side effects and contrasting efficiency. In this context, gut microbiota has emerged as a new target to modulate host metabolism including cholesterolemia. Coprostanol, a very low-absorbed derivative of cholesterol, is found in large amounts in the stools of individuals called high-converters (HC) and in small quantities in so-called low-converters (BC) individuals. However, the impact of this gut bacterial metabolism on blood cholesterol remains unclear. Only one cholesterol-converting bacterial strain has been isolated from human feces. Our objective is the study of bacteria converting cholesterol into coprostanol and their potential effect on host cholesterol metabolism. We isolated 27 new cholesterol-converting bacteria from the feces of HC subjects using different cholesterol supplemented culture media. The screening of genomic libraries constructed from two of them allowed the identification of clones of interest. The analysis of their DNA sequence should allow the identification of gene(s) involved in the metabolic pathway of conversion of cholesterol to coprostanol and unknown until now. Six bacteria were selected on their coprostanol production features, medium adaptability and biomass production. One was retained for its abilities: i) to colonize germ-free mice gastro-intestinal tract and ii) to produce high level of coprostanol in vivo. Germ-free mice associated with this bacterial strain or with a strain which does not produce coprostanol in vitro received a cholesterol enriched diet for 12 weeks. Cholesterolemia and in vivo production of coprostanol have been followed as well as expression of genes involved in cholesterol metabolism and inflammation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#2d9359"><a href="https://fems-microbiology.org/" rel="external nofollow noopener" target="_blank">FEMS</a></abbr></div> <div id="Potiron2017" class="col-sm-8"> <div class="title">Cholesterol conversion into coprostanol by bacterial strains isolated from the human gut microbiota can modify host cholesterol metabolism</div> <div class="author"> <em>A. Potiron</em>, <a href="https://fr.linkedin.com/in/aicha-kriaa-651679169" rel="external nofollow noopener" target="_blank">A. Kriaa</a>, S. Boudebbouze, A. Bruneau, M. Lhomme, P. Lesnik, <a href="https://fr.linkedin.com/in/emmanuelle-maguin-6b65922a" rel="external nofollow noopener" target="_blank">E. Maguin</a>, <a href="https://fr.linkedin.com/in/moez-rhimi-528a875a" rel="external nofollow noopener" target="_blank">M. Rhimi</a>, and <a href="https://fr.linkedin.com/in/philippe-g%C3%A9rard-5784b032" rel="external nofollow noopener" target="_blank">P. Gérard</a> </div> <div class="periodical"> <em>7th Congress of European Microbiologists - FEMS</em>, Jul 2017 </div> <div class="periodical"> Valencia, Spain </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li> </ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#00b5ff"><a href="https://www.febs.org/our-members/the-tunisian-association-of-biological-sciences/" rel="external nofollow noopener" target="_blank">ATSB</a></abbr></div> <div id="Potironetal2015" class="col-sm-8"> <div class="title">New insight in cholesterol metabolism by human gut microbiota</div> <div class="author"> <em>A. Potiron</em>, <a href="https://fr.linkedin.com/in/aicha-kriaa-651679169" rel="external nofollow noopener" target="_blank">A. Kriaa</a>, T. Le Roy, <a href="https://tn.linkedin.com/in/h%C3%A9la-mkaouar-910b3380" rel="external nofollow noopener" target="_blank">H. Mkaouar</a>, S. Boudebbouze, P. Lesnik, <a href="https://fr.linkedin.com/in/philippe-g%C3%A9rard-5784b032" rel="external nofollow noopener" target="_blank">P. Gérard</a>, <a href="https://fr.linkedin.com/in/emmanuelle-maguin-6b65922a" rel="external nofollow noopener" target="_blank">E. Maguin</a>, and <a href="https://fr.linkedin.com/in/moez-rhimi-528a875a" rel="external nofollow noopener" target="_blank">M. Rhimi</a> </div> <div class="periodical"> <em>26th Forum of the Tunisian Association of Biological Sciences</em>, Mar 2015 </div> <div class="periodical"> Monastir, Tunisia </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li></ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#5c892f"><a href="https://ives-openscience.eu/ives-conference-series/macrowine/" rel="external nofollow noopener" target="_blank">IVES</a></abbr></div> <div id="Bezuidenhoutetal2014" class="col-sm-8"> <div class="title">Bridging the gap between the grapevine terpene synthase gene family and volatile terpenes found in the berry and wine</div> <div class="author"> I. Bezuidenhout, P. R. Young, D. Jacobson, S. J. Smit, <em>A. Potiron</em>, and M. Vivier</div> <div class="periodical"> <em>Macrowine 2014</em>, Sep 2014 </div> <div class="periodical"> Stellenbosch, South Africa </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Grapevine (Vitis vinifera) has an extensive terpene synthase gene (VvTPS) family. A large portion of this gene family putatively encodes for mono- and sesquiterpene synthases; these two classes of terpenes are known contributors to the aroma of grape berries and wine. It is also known that terpenes are labile, with certain environmental conditions resulting in rearrangement of terpene structures. Various grapevine enzymes can also modify terpenes to create additional diversity in the terpenic products found in the berry. The winemaking process adds another level of complexity as yeast can also cause some alteration to the volatile terpene profile. Ultimately the objective would be to connect the activity of VvTPSs to the terpenes detected in the grape berry, as well as the ensuing wine. Here we report a workflow to evaluate, through functional characterisation studies in an optimised yeast system, the specific terpene product profiles of VvTPSs. To have a more holistic and integrated view of this complex gene family, available gene expression data is targeted to be correlated to functionally characterised VvTPSs and to the volatile terpenes detected in the grape berry or wine. The overall aim is to create multiple networks wherein VvTPS expression, terpene products (and possible derivatives) of characterised VvTPSs and volatile terpenes detected in various grapevine cultivars are connected. The results could support a more targeted approach in grapevine breeding and for an improved profiling of various grapevine cultivars in terms of their volatile terpene production potential.</p> </div> </div> </div> </li></ol> </div> <h2>thesis</h2> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="potironphilo" class="col-sm-8"> <div class="title">Data, Measurement, and Causality: Challenges from Modern Microbiology</div> <div class="author"> <em>Aline Potiron</em> </div> <div class="periodical"> Oct 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://epub.jku.at/obvulihs/download/pdf/10565899" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li></ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="potiron:tel-01812929" class="col-sm-8"> <div class="title">Cholesterol conversion into coprostanol by bacteria from human gut microbiota and its impact cholesterolemia</div> <div class="author"> <em>Aline Potiron</em> </div> <div class="periodical"> Dec 2017 </div> <div class="periodical"> </div> <div class="links"> <a href="https://pastel.hal.science/tel-01812929/file/64723_POTIRON_2017_archivage.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2025 Aline Potiron. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: October 27, 2025. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>